serum-390/soen390-team16-godzilla
# Refactored the backend packages - more clear project structure

# Description

The old backend package structure was a bit of a mess. This commit cleans
everything up back there.

## Link to work item

Closes #95 

## How to Test

Make sure that the project builds as normal. Try to build the docker image:
```bash
cd docker
./build-and-run.sh
```
